test/API/CMakeLists.txt

```diff
@@ -161,109 +169,6 @@ if (HDF5_ENABLE_FORMATTERS)
   clang_format (HDF5_TEST_h5_api_test_FORMAT h5_api_test)
 endif ()
 
-if (HDF5_TEST_API_INSTALL)
-  install (
-    TARGETS
-      h5_api_test
-    EXPORT
-      ${HDF5_EXPORTED_TARGETS}
-    DESTINATION
-      ${HDF5_INSTALL_BIN_DIR}
-    PERMISSIONS
-      OWNER_READ OWNER_WRITE OWNER_EXECUTE GROUP_READ GROUP_EXECUTE WORLD_READ WORLD_EXECUTE
-    COMPONENT
-      tests
-  )
-endif ()
-
-#-----------------------------------------------------------------------------
-# Build the ported HDF5 test executables
-#-----------------------------------------------------------------------------
-foreach (api_test_extra ${HDF5_API_TESTS_EXTRA})
```

Comment:
This whole block can be removed since we no longer build testhdf5, etc. twice and instead use the existing executable